astropy/visualization/mpl_normalize.py

```diff
@@ -15,11 +15,6 @@
 try:
     import matplotlib  # pylint: disable=W0611
     from matplotlib.colors import Normalize
-
-    # On older versions of matplotlib Normalize is an old-style class
```

Comment:
It is unclear to me which older versions based on this comment. Is this new-style only matplotlib our minversion?

Reply:
Python 3 has only new-style, but on Python 2 the classes that don't inherit from `object` are old-style.